data_dictionary.md

```diff
@@ -54,10 +55,11 @@ The Activity Stream addon sends two distinct types of pings to the backend (HTTP
 |-----|-------------|:-----:|
 | `action` | Either `activity_stream_event` or `activity_stream_session`. | :one:
 | `addon_version` | The version of the Activity Stream addon. | :one:
-| `click_position` | The index of the element in the `source` that was clicked. | :one:
+| `action_position` | The index of the element in the `source` that was clicked. | :one:
 | `client_id` | An identifier for this client. | :one:
 | `event` | The type of event. Either ("click", "share", "delete", "more_items") | :one:
 | `load_reason` | Either ("newtab", "refocus", "restore") and is the reason the tab was focused. | :one:
+| `page` | Either ("newtab", "timeline"). | :one:
```

Comment:
What about the "/timeline/bookmarks" page?

Also, bonus points for moving `page` after `max_scroll_depth` to preserve the A-z of the table. (I guess same über-nit with `action_position` above).